packages/message-kit/src/lib/handlerContext.ts

```
@@ -25,25 +25,32 @@ import {
+  conversation!: V2Conversation;
   client!: Client;
   version!: "v2" | "v3";
-  v2client!: ClientV2;
+  v2client!: V2Client;
   skills?: SkillGroup[];
   members?: AbstractedMember[];
   admins?: string[];
   superAdmins?: string[];
   sender?: AbstractedMember;
+  awaitingResponse: boolean = false;
+  awaitedHandler: ((text: string) => Promise<boolean | void>) | null = null;
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)